index.html

```diff
@@ -114,10 +125,14 @@ <h1 class="html-1">Javascript</h1>
                 </article>
             </article>
             <article class="card-section">
-                <img src="./images/card3.png" alt="tonic-snapshot" class="tonic-img">
+
+                <img src="./images/tonic-img-1.png" alt="tonic-snapshot" class="tonic-img-mobile">
+
+                <img src="./images/snaposhot-1-desktop-p.png" alt="tonic-snapshot" class="tonic-img-desktop">
+                <!--<img src="./images/Snapshoot-Portfolio-4.png" alt="tonic-snapshot" class="tonic-img-desktop">-->
```

Comment:
- [ ] [ Optional ] As requested by a previous reviewer, please remove commented lines of code from your code base, this is because commented code makes your repo look unprofessional.